src/core/numdiff/action.cpp

```diff
@@ -1,7 +1,8 @@
 ///////////////////////////////////////////////////////////////////////////////
 // BSD 3-Clause License
 //
-// Copyright (C) 2018-2019, LAAS-CNRS, New York University, Max Planck Gesellshaft
+// Copyright (C) 2018-2020, LAAS-CNRS, University of Edinburgh, New York University,
+// Max Planck Gesellshaft
```

Comment:
Typo in German: "Gesellschaft" (c missing)

Reply:
Thanks for catching these typos